canvas_account_admin_tools/views.py

```diff
@@ -88,7 +88,16 @@ def dashboard_account(request):
     custom_canvas_membership_roles = request.LTI['custom_canvas_membership_roles']
 
     """
-    Verify that the curernt user has permission to see the cross listing button
+    Verify that the current user has permission to see the Search Courses (aka course_info) tool
+    """
+    search_courses_allowed = is_allowed(
```

Comment:
Is there a place to keep release information, i.e. how we'd like to seed the various databases? Or is that already done in dev, qa, and prod? (Given this is a new permission, I would think it's safe to include it anytime before release, right?)

Reply:
We don't have any established place to keep that kind of information. I'll take care of updating the database before deployment.